### PR TITLE
Update pulumiverse plugin list

### PR DESCRIPTION
**Pull request by Frassle**
This adds a little generate script so this is easier to keep up-to date in the future.

### CODE REVIEW

**Review by julienp — APPROVED** [review]
Love it !

File: pkg/util/generate/templates/pulumiverse_list.go.template [hunk 1]
```diff
@@ -0,0 +1,22 @@
+// Copyright 2016-2024, Pulumi Corporation.
```
Comment (lunaris) [inline pkg/util/generate/templates/pulumiverse_list.go.template:1]:
```suggestion
// Copyright 2024, Pulumi Corporation.
```

File: pkg/util/generate/main.go [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+	"os"
+	"os/exec"
+	"path/filepath"
+	"slices"
+	"strings"
+
+	"github.com/google/go-github/v55/github"
+)
+
+func main() {
```
Comment (lunaris) [inline pkg/util/generate/main.go:16]: Maybe a quick doc string or `README.md` to say what this is doing?

**Review by lunaris — APPROVED** [review]
Siiiick

**Pull request merged**